recipe/meta.yaml

```diff
@@ -20,6 +20,7 @@ requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - scikit-build-core >=0.4.4
```

Comment:
I think `scikit-build-core` needs to be a host dependency (not build).

Reply:
Thanks for catching that. Looks good now.

Reply:
Thanks for catching that. Looks good now.